src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1907,62 +1826,75 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
         Set<String> studentsEmailInTeam = new HashSet<String>();
         if (role == UserRole.STUDENT) {
             student = studentsLogic.getStudentForEmail(courseId, userEmail);
-            List<StudentAttributes> studentsInTeam = studentsLogic
-                    .getStudentsForTeam(student.team, courseId);
+            List<StudentAttributes> studentsInTeam = studentsLogic.getStudentsForTeam(student.team, courseId);
             for (StudentAttributes teammates : studentsInTeam) {
                 studentsEmailInTeam.add(teammates.email);
             }
         }
-        
+
         InstructorAttributes instructor = null;
         if (role == UserRole.INSTRUCTOR) {
             instructor = instructorsLogic.getInstructorForEmail(courseId, userEmail);
         }
         
         Map<String, FeedbackResponseAttributes> relevantResponse = new HashMap<String, FeedbackResponseAttributes>();
```

Comment:
😂 Rewriting the entire method before refactoring isn't too feasible though, right? I might accidentally miss an edge case and the tests don't catch it. How would you suggest reorganising this method? I'm rewriting some blocks to increase readability in the process of refactoring, maybe we can start from there and slowly morph the method